services/galley/schema/src/V24.hs

```diff
@@ -0,0 +1,14 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes       #-}
+
+module V24 (migration) where
+
+import Cassandra.Schema
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 24 "Use LeveledCompactionStrategy on clients" $ do
+    schema' [r|
+        ALTER TABLE clients
+            WITH compaction = {'class': 'LeveledCompactionStrategy'}
```

Comment:
I think the statement is missing a `;` after the last `}` 😀

Reply:
Oh, good catch.